### DISCUSSION TITLE
Serverless setup?

### DISCUSSION THREAD

**Discussion by anlesk** [philosowaffle/peloton-to-garmin#70] [Ideas]
What do you think of moving the setup from the docker to serverless/lambda functions?
I'm thinking of the best way to automate the process but stick to a free cloud plan.

Basically, I can trigger the import automatically by calling the API provided by lambdas.

**Comment by philosowaffle** [comment]
It's been many many years since I last worked with lambda.  In theory this should work and would be an interesting option to pursue.  Even so, I think we will always need to support some form of our current version for the less tech-savvy, and for those who prefer to self-host.

But if we can support both deployment methods side by side with minimal effort then go for it.
